File: tournaments/stolenLunch/stolenLunch.py
def stolenLunch(note):
    newNote = []

    for i in range(0, len(note)):
        if '0' <= note[i] <= '9':
            newNote.append(chr(int(note[i]) + ord('a')))
        elif 'a' <= note[i] <= 'j':
            newNote.append(str(ord(note[i]) - ord('a')))
        else:
            newNote.append(note[i])
            
    return "".join(newNote)

File: tournaments/stolenLunch/stolenLunch.pl
#!/usr/bin/perl
use strict;
use warnings FATAL => 'all';

sub stolenLunch {
    my ($note) = @_;

    my @store = split('', $note);
    my $n = @store;
    my @result = (0)x$n;

    for (my $i = 0; $n > $i; ++$i) {
        my $item = $store[$i];
        if ('0' le $item && '9' ge $item) {
            $item = chr(ord($item) - ord('0') + ord('a'));
        } elsif ('a' le $item && 'j' ge $item) {
            $item = ord($item) - ord('a');
        }
        $result[$i] = $item;
    }

    return join('', @result);
}
